### PR TITLE
Finger driver

### PR DESCRIPTION
**Pull request by thiagodefreitas**
@ipa-fmw, @ipa-nhg . Please review

### CODE REVIEW

File: schunk_finger/package.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package>
+  <name>schunk_finger</name>
+  <version>0.0.1</version>
+  <description>The schunk_finger package</description>
+
+  <maintainer email="tdf@ipa.fhg.de">Thiago de Freitas</maintainer>
+  <license>BSD</license>
```
Comment (floweisshardt) [inline schunk_finger/package.xml]: please change to LGPL

File: schunk_finger/src/finger_driver.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    self._ser.close()
+  
+  def execute_cb(self, goal):
+    
+    position_distal = goal.trajectory.points[-1].positions[0]
+    position_proximal = goal.trajectory.points[-1].positions[1]
+    
+    self.move(position_distal, position_proximal)
+    
+    success = True
```
Comment (floweisshardt) [inline schunk_finger/src/finger_driver.py]: this is always true. I there a way to know if the finger really reached the goal or not?

**Comment by floweisshardt** [comment]
does the "schunk_finger" have an original Schunk name (e.g. like PRL60, PG80, ...). If so, we should use this name in the package name instead of "schunk_finger"

**Comment by thiagodefreitas** [comment]
Updated

**Comment by floweisshardt** [comment]
visual review: OK
tested on SDHx on TDF table

**Pull request merged**